ksqldb-tools/src/test/java/io/confluent/ksql/tools/migrations/MigrationsTest.java

```diff
@@ -229,6 +234,16 @@ private static void verifyMigrationsApplied() {
     assertThat(current.get(1).getRow().get().getColumns().get(2), is("bar bar BAR"));
     assertThat(current.get(1).getRow().get().getColumns().get(3), is("MIGRATED"));
     assertThat(current.get(1).getRow().get().getColumns().get(7), is("1"));
+
+    // verify foo
+    final List<StreamedRow> foo = assertThatEventually(
+        () -> makeKsqlQuery("SELECT * FROM FOO EMIT CHANGES LIMIT 2;"),
+        hasSize(4));
```

Comment:
nit: add a comment explaining why size 4. Is it because there's a header and a footer?